subsys/net/ip/ipv6.h

```diff
@@ -641,7 +641,7 @@ static inline int net_ipv6_pe_init(struct net_if *iface)
 #endif /* CONFIG_NET_IPV6_PE */
 
 typedef void (*net_ipv6_pe_filter_cb_t)(struct in6_addr *prefix,
-					bool is_blacklist,
+					bool is_denylist,
```

Comment:
Any particular reason for using denylist, while using blocklist in the other occurrences?

Reply:
Not really, except I found this "client" in-tree 
https://github.com/zephyrproject-rtos/zephyr/blob/c031e127b0fde2f538113f60ff530743da3cbba2/subsys/net/lib/shell/ipv6.c#L60 and aimed for consistency

Reply:
Got it. Consistency is good.